awx/api/views/mixin.py

```diff
@@ -46,8 +42,6 @@ class ActivityStreamEnforcementMixin(object):
     '''
     def check_permissions(self, request):
```

Comment:
We can probably get rid of this entire `check_permissions` method, because the only reason it exists is to perform the extra `feature_enabled` called.